File: market-data-public/leverage-tiers/leverage-tiers.py
import requests

def fetch_leverage_tiers(market_code):
    url = f'https://api.ox.fun/v3/leverage/tiers?marketCode={market_code}'

    try:
        response = requests.get(url)
        response.raise_for_status()
        print('Leverage Tiers Data:', response.json())
    except requests.exceptions.RequestException as error:
        print('Error fetching leverage tiers data:', error)

# Example usage
fetch_leverage_tiers('BTC-USD-SWAP-LIN')

File: Rest-Api-V3/market-data-public/leverage-tiers/leverage-tiers.js
const axios = require('axios');

async function fetchLeverageTiers(marketCode) {
  const url = `https://api.ox.fun/v3/leverage/tiers?marketCode=${marketCode}`;

  try {
    const response = await axios.get(url);
    console.log('Leverage Tiers Data:', response.data);
  } catch (error) {
    console.error('Error fetching leverage tiers data:', error.response ? error.response.data : error.message);
  }
}

// Example usage
fetchLeverageTiers('BTC-USD-SWAP-LIN');